tiled/server/router.py

```diff
@@ -556,11 +556,25 @@ def post_metadata(
             specs=body.specs,
         )
     else:
-        raise HTTPException(status_code=404, detail="Failed to write in this node.")
+        raise HTTPException(status_code=404, detail="This path cannot accept metadata.")
 
     return json_or_msgpack(request, {"key": key})
 
 
+@router.delete("/node/delete/{path:path}")
+async def delete(
+    request: Request,
+    entry=Security(entry, scopes=["write:data", "write:metadata"]),
+):
+    if hasattr(entry, "delete"):
+        entry.delete()
+    else:
+        raise HTTPException(
+            status_code=404, detail="This path does not support deletion."
```

Comment:
Method not allowed (405) would be better here.

Reply:
is that true for the other writing routes as well?

Reply:
Ah, it is! I didn’t think of this at first because the original toy implementation added new routes. Mind updating while you are here?

Reply:
updated all of them